### PR TITLE
maybe fix loading corrupted data

### PR DESCRIPTION
**Pull request by leoscholl**

### CODE REVIEW

**Comment by leoscholl** [comment]
code to delete task data from an existing hdf file:

```
# Make a dummy HDF file by copying an existing HDF file from the same experiment
import shutil
files['hdf'] = 'dummy_hdf_rig1_center_out.hdf'
shutil.copy(os.path.join(data_dir, 'beig20230109_15_te7977.hdf'), os.path.join(data_dir, files['hdf']))
with tables.open_file(os.path.join(data_dir, files['hdf']), 'r+') as f:
    f.get_node('/task').truncate(0)
```

File: tests/test_preproc.py [hunk 1]
```diff
@@ -903,6 +904,27 @@ def test_parse_bmi3d_v13(self):
...
+
+        # Test if a dummy HDF file is provided
+        files['hdf'] = 'dummy_hdf_rig1_v13.hdf'
+
+        # Make a dummy HDF file by copying an existing HDF file from the same experiment
+        import shutil
+        shutil.copy(os.path.join(data_dir, 'beig20230109_15_te7977.hdf'), os.path.join(data_dir, files['hdf']))
+        with tables.open_file(os.path.join(data_dir, files['hdf']), 'r+') as f:
+            f.get_node('/task').truncate(0)
+
```
Comment (rcanfield2) [inline tests/test_preproc.py:922]: Do we need to make a wrapper to do this?

**Review by rcanfield2 — APPROVED** [review]

File: tests/test_preproc.py (hunk 1 above)
Comment (leoscholl) [reply tests/test_preproc.py:922]: Yeah something that wraps around proc_exp() maybe?

Comment (rcanfield2) [reply tests/test_preproc.py:922]: yeah, something like proc_exp_new_hdf()

**Pull request merged**